transports/transport-highway/src/main/java/io/servicecomb/transport/highway/HighwayClient.java

```diff
@@ -100,22 +105,26 @@ public void send(Invocation invocation, AsyncResponse asyncResp) throws Exceptio
     tcpClientPool.send(tcpClient, clientPackage, ar -> {
       // 此时是在网络线程中，转换线程
       invocation.getResponseExecutor().execute(() -> {
-        if (ar.failed()) {
-          // 只会是本地异常
-          asyncResp.consumerFail(ar.cause());
-          return;
-        }
-
-        // 处理应答
         try {
-          Response response =
-              HighwayCodec.decodeResponse(invocation,
-                  operationProtobuf,
-                  ar.result(),
-                  tcpClient.getProtobufFeature());
-          asyncResp.complete(response);
-        } catch (Throwable e) {
-          asyncResp.consumerFail(e);
+          if (ar.failed()) {
+            // 只会是本地异常
+            asyncResp.consumerFail(ar.cause());
+            return;
+          }
+
+          // 处理应答
+          try {
+            Response response =
+                HighwayCodec.decodeResponse(invocation,
+                    operationProtobuf,
+                    ar.result(),
+                    tcpClient.getProtobufFeature());
+            asyncResp.complete(response);
+          } catch (Throwable e) {
+            asyncResp.consumerFail(e);
+          }
+        } finally {
+          invocation.triggerFinishedEvent();
```

Comment:
in reactive mode, maybe the "finished" time include next request time
normally there is no problem, because reactive task is very fast
but it's better not to include that time.

Reply:
response complete code is written by user, i seems difficult to separate